ios/Social Distance Alarm/Social Distance Alarm/Info.plist

```diff
@@ -50,8 +50,6 @@
 	<key>UIBackgroundModes</key>
 	<array>
 		<string>audio</string>
-		<string>bluetooth-central</string>
```

Comment:
I'm not exactly clear about this change but if it doesn't cause any regression it's fine.

Reply:
We don't use background mode, so I believe this may fix some odd behavior.